ebu_tt_live/scripts/ebu_ebuttd_converter.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from argparse import ArgumentParser
+from .common import create_loggers
+
+from ebu_tt_live.node import EBUTTDConverterConsumer
+from ebu_tt_live.clocks.local import LocalMachineClock
+from ebu_tt_live.twisted import TwistedConsumer, BroadcastClientFactory, ClientNodeProtocol
+from ebu_tt_live.carriage.twisted import TwistedConsumerImpl
+from twisted.internet import task, reactor
+
+
+log = logging.getLogger('ebu_simple_consumer')
+
+
+parser = ArgumentParser()
+
+parser.add_argument('-c', '--config', dest='config', metavar='CONFIG')
+parser.add_argument('-i', '--interval', dest='interval', metavar='INTERVAL', type=float, default=2.0)
+
+
+def main():
+    args = parser.parse_args()
+    create_loggers()
+    log.info('This is a Simple Consumer example')
+
+
+    consumer_impl = TwistedConsumerImpl()
```

Comment:
Can be done later, but we should add the file system impl back in here so we can do this repeatably with stored files.